services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -1837,3 +1837,59 @@ func (u *UndeployApplication) Transform(
 	}
 	return fmt.Sprintf("application '%v' was deleted successfully", u.Application), nil
 }
+
+type EvtCreateEnvironmentGroupLock struct {
```

Comment:
Isn't it better to name it `CreateEnvironmentGroupLock` instead?

Reply:
done